packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -7,12 +7,15 @@ import debug from '../../debug'
 
 const helpText = `
 Options
-  --wait Waits for the operation to finish
+  --background Start the copy and return without waiting for it to finish
+  --progress <job-id> Show progress for a running copy
 
 Examples
   sanity dataset copy
   sanity dataset copy <source-dataset>
   sanity dataset copy <source-dataset> <target-dataset>
+  sanity dataset copy --background <source-dataset> <target-dataset>
+  sanity dataset copy --progress <job-id>
```

Comment:
Not sure if we should add this as an option, as `copy` sounds like a command, but when adding this flag, it doesn't do that anymore, only listen on an existing copy operation. Maybe this progress thing should be a part of a top-level API in the CLI?

Reply:
I like to think of `dataset` as the top-level "category" and then "copy" as the category below it, and that it therefore makes okay sense to have it here. I feel that moving it to a different top-level thing could make it more difficult to find doc-wise.

However I also recognise that this is not a good "UNIX"-way of doing it. So if we think it's worth spending time on moving into a top-level category then I'd love to do it 🙂 Just let me know!

Reply:
I think having a seperate subcommand to track progress of "any job" may also be an option here: `sanity job listen <job-id>`. This is not the only job which CLI will ever listen to, we may have an export to bucket job tomorrow which is triggered with `sanity dataset export ...` and can be tracked with `sanity job listen ...`. I would say this is worth discussing in the team.

Reply:
> I think having a seperate subcommand to track progress of "any job" may also be an option here: `sanity job listen <job-id>`. This is not the only job which CLI will ever listen to, we may have an export to bucket job tomorrow which is triggered with `sanity dataset export ...` and can be tracked with `sanity job listen ...`. I would say this is worth discussing in the team.

Thanks for the input @j33ty! 🙂 We've discussed this within the team and found that this is a good enough solution for now because the only thing we have which uses it at this point is the copy command and it doesn't state verbosely that this is a job, just that you can attach/detach to a copy.

But when we get more opportunities to use listen we'll move it to be a top level command 🙂